### PR TITLE
[Just] Migrates start-test to just

### PR DESCRIPTION
**Pull request by kenotron**
#### Pull request checklist

- [x] Include a change request file using `$ npm run change`

#### Description of changes

One of the left over tasks that hadn't been migrated is start-test. This moves it to use just.

#### Focus areas to test

(optional)

### CODE REVIEW

File: scripts/tasks/jest.js [hunk 1]
```diff
@@ -9,3 +9,11 @@ exports.jest = () =>
     ...(process.env.TRAVIS || argv().production ? { coverage: true } : undefined),
     ...(argv().u || argv().updateSnapshot ? { updateSnapshot: true } : undefined)
   });
+
+exports.jestWatch = () =>
+  jestTask({
+    ...(process.env.TRAVIS && { runInBand: true }),
+    ...(process.env.TRAVIS || argv().production ? { coverage: true } : undefined),
```
Comment (cliffkoh) [inline scripts/tasks/jest.js]: just wondering why you just use `&&` in line 15 and then ternary in line 16 and 17..

Comment (kenotron) [reply scripts/tasks/jest.js]: ah, no reason... perhaps to confuse and amaze.

**Comment by cliffkoh** [comment]
Also have to do the change in #7533's new package.json

**Comment by kenotron** [comment]
I don't think they added start-test in their package.json (yet)

**Comment by kenotron** [comment]
Seems like the lightrail service isn't working?

**Pull request merged**